From: Debian Java Maintainers <pkg-java-maintainers@lists.alioth.debian.org>
Date: Sun, 12 Nov 2017 21:04:50 +0100
Subject: generate_pom

By default Gradle won't generate POMs for itself. generate_pom.gradle makes
Gradle auto generate POMs for all of the JARs so that we can install the POMs
to /usr/share/maven-repo

Forwarded: not-needed
---
 build.gradle | 1 +
 1 file changed, 1 insertion(+)

--- a/build.gradle
+++ b/build.gradle
@@ -216,6 +216,7 @@
         apply from: "$rootDir/gradle/java9.gradle"
         check.dependsOn ":docs:checkstyleApi"
         check.dependsOn "codeQuality"
+        apply from: "${rootDir}/debian/generatePom.gradle"
     }
 
     if (project in publishedProjects) {
